msal/src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -148,23 +179,33 @@ public async Task<IEnumerable<IAccount>> GetAccountsAsync()
         }
 
         /// <summary>
-        /// Get the account by identifier from the accounts available in the cache.
+        /// Get the <see cref="IAccount"/> by its identifier among the accounts available in the token cache.
         /// </summary>
-        /// <param name="accountId">account identifier</param>
+        /// <param name="accountId">account identifier. The value of the identifier will probably have been stored value from the
```

Comment:
>a [](start = 36, length = 1)

general comment, I think there are a number of places where you use small letter after the param, not sure that is intentional. IMO, it should be capitalized.